src/userscores/internal/exportscoreservice.h

```diff
@@ -0,0 +1,72 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#ifndef MU_USERSCORES_EXPORTSCORESERVICE_H
+#define MU_USERSCORES_EXPORTSCORESERVICE_H
+
+#include "modularity/ioc.h"
+
+#include "iexportscoreservice.h"
+#include "iuserscoresconfiguration.h"
+#include "iinteractive.h"
+#include "notation/inotationwritersregister.h"
+#include "importexport/imagesexport/iimagesexportconfiguration.h"
+#include "context/iglobalcontext.h"
+#include "notation/inotation.h"
+#include "system/ifilesystem.h"
+
+namespace mu::userscores {
+class ExportScoreService : public IExportScoreService
```

Comment:
better rename this class to ExportScoreScenario because in this class you interact with the user, not only get set some data